packages/cloudflare/src/cli/build/patches/ast/vercel-og.ts

```diff
@@ -0,0 +1,63 @@
+import { SgNode } from "@ast-grep/napi";
+
+import { applyRule } from "./util.js";
+
+export const vercelOgImportRule = `
+rule:
+  pattern: $NODE
+  kind: string
+  regex: "next/dist/compiled/@vercel/og/index.node.js"
+inside:
+  kind: arguments
+  inside:
+    kind: call_expression
+    stopBy: end
+    has:
+      field: function
+      regex: "import"
+
+fix: |-
+  "next/dist/compiled/@vercel/og/index.edge.js"
+`;
+
+/**
+ * Patches Node.js imports for the library to be Edge imports.
+ *
+ * @param root Root node.
+ * @returns Results of applying the rule.
+ */
+export function patchVercelOgImport(root: SgNode) {
+  return applyRule(vercelOgImportRule, root);
+}
+
+export const vercelOgFallbackFontRule = `
+rule:
+  kind: variable_declaration
+  all:
+    - has:
+        kind: variable_declarator
+        has:
+          kind: identifier
+          regex: ^fallbackFont$
```

Comment:
nit: maybe it would be safer to detect the `fetch(font_name)` in case the variable name changes - but let's merge this, we can refine later if needed.